src/app/components/horse-books-listing/horse-book-list-item/horse-book-list-item.component.scss

```diff
@@ -21,19 +19,28 @@ mat-card{
 	text-align: middle;
 }
 .image img{
+	padding-top: 10%;
```

Comment:
Is this necessary for the fix?

Reply:
the padding-top? That was in there before.

Reply:
This is a diff between master and this branch.